forui/lib/src/widgets/tabs/tabs.dart

```diff
@@ -0,0 +1,167 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import 'package:flutter_localizations/flutter_localizations.dart';
+
+import 'package:forui/forui.dart';
+
+part 'tabs_style.dart';
+
+part 'tab_controller.dart';
+
+/// An object that represents a tab entry in a group of tabs.
+class FTabEntry {
```

Comment:
The design seems a little weird. The string label is not exposed. Instead a `label` getter is provided which returns either a public `rawLabel` or a wrapped `label`. I think it'll be better if `FTabEntry` exposed the string label and rawLabel. The logic for deciding which label to use can be moved to `FTabs`.

Reply:
I have pushed new changes, need your help to check if the logic makes sense. Line 113 below.